test/TestApps/PushNotificationsTestApp/main.cpp

```diff
@@ -280,49 +280,36 @@ void UnregisterClsid()
 int main()
 {
     initUnitTestMapping();
-    bool output = false;
+    bool failed = false;
```

Comment:
At least for me this variable name is confusing.

If a test passes, `runUnitTest()` returns true. So `failed` is `true`? Does this mean the test failed?

Reply:
The RunUnitTest function returns a bool and I didn't want to write an if/else check to return 0 or 1 to main. So !failed means that the test returned true (1 -> 0). Returning exit code 0 to main is supposed to mean success.

Reply:
I see. You can rename `failed` as `testResult` and do `return testResult ? 0 : 1`;